### ISSUE TITLE
Duplicate Anchor Component

### ISSUE THREAD

**Issue by furti** [porscheinformatik/selenium-components#2]
Why is there a at.porscheinformatik.seleniumcomponents.component.LinkComponent and a at.porscheinformatik.seleniumcomponents.component.AComponent for the same thing, a HTML anchor element?

AComponent is not the best name for a component btw. because it looks so generic.
Given that SeleniumContainer uses a generic type of ```AnyChildComponent``` the name ```AComponent``` is really confusing.

**Comment by thred** [comment]
Yep. Had my problems with the name, too. On one hand we could name components like the HTML tags. The name "AComponent" is correct in this case. On the other hand, we could name the components more generic, like "LinkComponent", "HtmlComponent", "BlockComponent". First, I thought it's better to name the elements like the HTML tags for a couple of reasons, e.g. that you can skip the selector or build special components for Angular. But the more I use these component, it seems better to build and use generic components. Currently I would remove the explicitly named components and just use generic ones. Ok?
